apps/storybook/src/TiledHeatmapMesh/TiledHeatmapMesh.stories.tsx

```diff
@@ -0,0 +1,207 @@
+import type { AxisConfig, Domain, TiledHeatmapMeshProps } from '@h5web/lib';
+import {
+  DefaultInteractions,
+  TiledHeatmapMesh,
+  TiledTooltipMesh,
+  VisCanvas,
+} from '@h5web/lib';
+import { ScaleType } from '@h5web/shared';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import FillHeight from '../decorators/FillHeight';
+import { CheckerboardTilesApi } from './checkerboard-api';
+import LinearAxesGroup from './LinearAxesGroup';
+import { MandelbrotTilesApi } from './mandlebrot-api';
+import { renderTooltip } from './utils';
+
+interface TiledHeatmapStoryProps extends TiledHeatmapMeshProps {
+  abscissaConfig: AxisConfig;
+  ordinateConfig: AxisConfig;
+}
+
+const defaultApi = new MandelbrotTilesApi(
+  { width: 1e9, height: 1e9 },
+  { width: 128, height: 128 },
+  [-2, 1],
+  [-1.5, 1.5]
+);
+const halfMandelbrotApi = new MandelbrotTilesApi(
+  { width: 1e9, height: 5e8 },
+  { width: 256, height: 128 },
+  [-2, 1],
+  [0, 1.5]
+);
+const checkerboardApi = new CheckerboardTilesApi(
+  { width: 1e9, height: 1e9 },
+  { width: 128, height: 128 }
+);
+
+const meta = {
+  title: 'Experimental/TiledHeatmapMesh',
+  component: TiledHeatmapMesh,
+  decorators: [FillHeight],
+  parameters: {
+    layout: 'fullscreen',
+    controls: { sort: 'requiredFirst' },
+  },
+  args: {
+    invertColorMap: false,
+    colorMap: 'Viridis',
+    domain: [0, 1],
+    scaleType: ScaleType.Linear,
+    displayLowerResolutions: true,
+    qualityFactor: 1,
+  },
+  argTypes: {
+    scaleType: {
+      control: { type: 'inline-radio' },
+      options: [
+        ScaleType.Linear,
+        ScaleType.Log,
+        ScaleType.SymLog,
+        ScaleType.Sqrt,
+      ],
+    },
+    qualityFactor: {
+      control: { type: 'range', min: 0, max: 1, step: 0.1 },
+    },
+  },
+} satisfies Meta<TiledHeatmapStoryProps>;
+
+export default meta;
+type Story = StoryObj<TiledHeatmapStoryProps>;
+
+export const Default = {
+  render: (args) => {
+    const { abscissaConfig, ordinateConfig, ...tiledHeatmapProps } = args;
+
+    return (
+      <VisCanvas
+        abscissaConfig={abscissaConfig}
+        ordinateConfig={ordinateConfig}
+        aspect="equal"
+      >
+        <DefaultInteractions />
+        <group
+          scale={[
+            abscissaConfig.flip ? -1 : 1,
+            ordinateConfig.flip ? -1 : 1,
+            1,
+          ]}
+        >
+          <TiledHeatmapMesh {...tiledHeatmapProps} />
+        </group>
+        <TiledTooltipMesh renderTooltip={renderTooltip} />
```

Comment:
I've removed the extra `showTooltip` and `showAxes` args. I'm now always showing the tooltips and I've removed the `NoAxes` story (since this is already documented by a dedicated `VisCanvas` story).